docs/getting-started.md

```diff
@@ -1 +1,106 @@
-This is a placeholder for some docs
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on Kubernetes.
+It has great powers, but deploying it may not be so easy, depending on how and where you deploy
+your Kubernetes. This tutorial will show you an easy way to deploy Kubeflow using MicroK8s, a
+lightweight version of Kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy MicroK8s on Ubuntu
+* How to deploy Kubeflow using MicroK8s
+* How to access your Kubeflow dashboard
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 16GB RAM, 60GB Disk)
+* Some basic command-line knowledge
+* If installing on Windows or MacOS, see the MicroK8s docs for OS-specific requirements:
+  <https://microk8s.io/docs/install-alternatives> 
+
+
+## 2. Install MicroK8s
```

Comment:
I would change this section to simply say:

"To install MicroK8s, follow the installation instructions for your platform at https:/microk8s.io/docs."

And then, I'd delete section 3 (Join admin group) entirely.